pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "framegrab"
-version = "0.4.6"
+version = "0.4.7"
```

```suggestion
version = "0.5.0"
```
Exposing new options justifies a major version bump, even though it's technically backwards compatible.